File: java/src/main/java/gherkin/pickles/PickleLocation.java
package gherkin.pickles;

public class PickleLocation {
    private final String path;
    private final int line;
    private final int column;

    public PickleLocation(String path, int line, int column) {
        this.path = path;
        this.line = line;
        this.column = column;
    }

    public String getPath() {
        return path;
    }

    public int getLine() {
        return line;
    }

    public int getColumn() {
        return column;
    }
}

File: dotnet/Gherkin/Pickles/PickleLocation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Gherkin.Pickles
{
    public class PickleLocation
    {
        public string Path { get; private set; }
        public int Line { get; private set; }
        public int Column { get; private set; }

        public PickleLocation(string path, int line, int column)
        {
            Path = path;
            Line = line;
            Column = column;
        }
    }
}
